src/dscanner/analysis/base.d

```diff
@@ -139,11 +139,23 @@ extern(C++) class BaseAnalyzerDmd(AST) : ParseTimeTransitiveVisitor!AST
 
 protected:
 
+	bool inAggregate;
+
 	extern(D) void addErrorMessage(size_t line, size_t column, string key, string message)
 	{
 		_messages.insert(Message(fileName, line, column, key, message, getName()));
 	}
 
+	template visitTemplate(T)
```

Comment:
This also seems unneeded.

Reply:
I added them because they are in the actual implementation and I assumed we will eventually need them down the road. Should I delete and then add them when they will be required?

Reply:
Yes.

Reply:
We incrementally add things when we need them, otherwise, we might end up adding code that we will never use.